zio-http/src/main/scala/zhttp/http/Response.scala

```diff
@@ -81,8 +81,9 @@ final case class Response private (
       case HttpData.BinaryByteBuf(data) => data
       case HttpData.BinaryStream(_)     => null
       case HttpData.Empty               => Unpooled.EMPTY_BUFFER
-      case HttpData.File(file)          =>
-        jHeaders.set(HttpHeaderNames.CONTENT_TYPE, Files.probeContentType(file.toPath))
+      case HttpData.File(raf, path)     =>
+        jHeaders.set(HttpHeaderNames.CONTENT_TYPE, Files.probeContentType(path))
+        jHeaders.set(HttpHeaderNames.CONTENT_LENGTH, raf.length())
```

Comment:
Can probing and content-length be added on the frontend?

Reply:
Implemented using middleware, in `fromPath`